### PR TITLE
FIX: Unify BrainVision reading

### PR DESCRIPTION
**Pull request by Eric89GXL**
Ready for review @teonlamont

### CODE REVIEW

File: mne/io/brainvision/brainvision.py [hunk 1]
```diff
@@ -420,7 +306,7 @@ def _get_eeg_info(vhdr_fname, eog, misc, response_trig_shift):
     if settings.find('[Comment]') != -1:
         params, settings = settings.split('[Comment]')
     else:
-        params, settings = settings, str()
+        params, settings = settings, ''
```
Comment (teonlamont) [inline mne/io/brainvision/brainvision.py]: shouldn't this stay str() to be explicit or would that cause complications for str vs stringtype?

Comment (Eric89GXL) [inline mne/io/brainvision/brainvision.py]: I don't think we use `str()` in many other places in the codebase, so I slightly prefer `''` for consistency.

File: mne/io/brainvision/brainvision.py [hunk 2]
```diff
@@ -429,57 +315,44 @@ def _get_eeg_info(vhdr_fname, eog, misc, response_trig_shift):
 
     # get sampling info
     # Sampling interval is given in microsec
-    sfreq = 1e6 / cfg.getfloat('Common Infos', 'SamplingInterval')
-    sfreq = int(sfreq)
-    eeg_info['n_eeg_chan'] = n_eeg_chan = cfg.getint('Common Infos',
-                                                     'NumberOfChannels')
+    sfreq = int(1e6 / cfg.getfloat('Common Infos', 'SamplingInterval'))
```
Comment (teonlamont) [inline mne/io/brainvision/brainvision.py]: just wondering, so what's the verdict on `sfreq`? do we store it as an `int` or `float`?

Comment (Eric89GXL) [inline mne/io/brainvision/brainvision.py]: ahh crap, it's supposed to be a float

File: mne/io/brainvision/brainvision.py [hunk 3]
```diff
@@ -602,11 +471,11 @@ def read_raw_brainvision(vhdr_fname, montage=None,
...
         Names of channels or list of indices that should be designated
         EOG channels. Values should correspond to the vhdr file
-        Default is ('HEOGL', 'HEOGR', 'VEOGb').
+        Default is ``('HEOGL', 'HEOGR', 'VEOGb')``.
     misc : list or tuple of str
         Names of channels or list of indices that should be designated
         MISC channels. Values should correspond to the electrodes
-        in the vhdr file. Default is None.
+        in the vhdr file. Default is ``()``.
     reference : None | str
```
Comment (teonlamont) [inline mne/io/brainvision/brainvision.py]: Add deprecation warning here too

File: mne/io/brainvision/brainvision.py [hunk 4]
```diff
@@ -602,11 +471,11 @@ def read_raw_brainvision(vhdr_fname, montage=None,
...
         Name of the electrode which served as the reference in the recording.
```
Comment (teonlamont) [inline mne/io/brainvision/brainvision.py]: Add deprecation warning here too

**Comment by teonlamont** [comment]
that's it for me

**Comment by agramfort** [comment]
merge when you're happy. No time to look.

**Comment by Eric89GXL** [comment]
Will do, should be today sometime

**Comment by Eric89GXL** [comment]
Thanks for the quick review @teonlamont

**Comment by teonlamont** [comment]
thanks @Eric89GXL!

**Pull request merged**